data.js

```diff
@@ -74,29 +73,53 @@ exports.data = [
             updatedBy: "Andre", updatedAt: "2019-08-18"
         },
         established: {
             content: "[1961](http://wwf.panda.org/knowledge_hub/history/)",
             updatedBy: "Andre", updatedAt: "2019-08-06"
         },
         kpis: {
-            content: "- Actions done / pa \n- Participants in actions\n- Members\n- Press Mentions",
+            content: "- Actions done / pa \n- Participants in actions\n- Members\n- Press mentions \n- advised organization \n- Visitors on Website",
             updatedBy: "Andre", updatedAt: "2019-08-06",
-
         },
+        actions: {
+            content: "These vary. With their expertise as a big environmental player they advise (countries, communities and private companies)[https://www.worldwildlife.org/initiatives/climate] in the topic."+
+                    "Own (press releases)[https://www.worldwildlife.org/press-releases?initiative_id=climate] are done."+
+                    "Research like (Climate Crowd)[https://www.wwfclimatecrowd.org/], which is trying to crowdsource information on the impact of climate change."+
+                    "Another focus to fight climate crisis is their (deforstation program.)[https://www.worldwildlife.org/initiatives/climate]",
+            updatedBy: "Andre", updatedAt: "2020-01-14",
+
         activeCountries: {
             content: "[about 100 countries](http://wwf.panda.org/knowledge_hub/history/50_years_of_achievements/)",
             updatedBy: "Andre", updatedAt: "2019-08-18",
         },
+        participation: {
+            content: "In their most reaching action the (World Hour)[https://www.worldwildlife.org/pages/earth-hour] over 12,000 landmarks and monuments switched off their lights for a whole hour and over 3 Billion social media posts had been done in about (this action in 2017)[http://awsassets.panda.org/downloads/Earth_Hour_Report_2017.pdf#page=6]",
+            updatedBy: "Andre", updatedAt: "2020-01-14"
+        },
         yearlyIncome: {
             content: "- 752M € in [2018](http://d2ouvy59p0dg6k.cloudfront.net/downloads/wwfintl_annualreview20182.pdf)" +
                 "\n- 145.9M € in [2013](http://d2ouvy59p0dg6k.cloudfront.net/downloads/ar2013_v15_lr_1.pdf)",
             updatedBy: "Andre", updatedAt: "2019-08-18"
         },
+
+        employees: {
+            content: "6,200 in (2015)[https://web.archive.org/web/20150206122603/http://wwf.panda.org/wwf_quick_facts.cfm]",
+            updatedBy: "Andre", updatedAt: "2019-08-06"
     },
-    {
+    members: {
+        content: "5+ M in (2015)[https://web.archive.org/web/20150206122603/http://wwf.panda.org/wwf_quick_facts.cfm]",
+        updatedBy: "Andre", updatedAt: "2020-01-14"
+    },
+        pressArticles: {
+            content: "They have a [press section](https://www.worldwildlife.org/press-mentions?year=2019) for external press.",
```

Comment:
"Their [press mentions section](https://www.worldwildlife.org/press-mentions?year=2019) is actively maintained."

Reply:
should had been done also on the 22.01...